### PR TITLE
Updated guides URLs

### PR DESCRIPTION
**Pull request by nodeMD**
## Purpose
https://github.com/luckyframework/lucky/issues/1498

## Description
Updated outdated guides URLs

## Checklist
* [x] - An issue already exists detailing the issue/or feature request that this PR fixes
* [x] - All specs are formatted with `crystal tool format spec src`
* [x] - Inline documentation has been added and/or updated
* [x] - Lucky builds on docker with `./script/setup`
* [x] - All builds and specs pass on docker with `./script/test`

### CODE REVIEW

File: tasks/gen/templates/resource/operations/{{operation_filename}}.cr.ecr [hunk 1]
```diff
@@ -1,5 +1,5 @@
 class <%= operation_class %> < <%= resource %>::SaveOperation
   # To save user provided params to the database, you must permit them
-  # https://luckyframework.org/guides/database/validating-saving#perma-permitting-columns
+  # https://luckyframework.org/guides/database/saving-records#perma-permitting-columns
```
Comment (nodeMD) [inline tasks/gen/templates/resource/operations/{{operation_filename}}.cr.ecr:3]: In PR: https://github.com/luckyframework/lucky/pull/1500
This file was renamed to save_{{underscored_resource}}.cr.ecr so after merging that PR the link in save_{{underscored_resource}}.cr.ecr will have to be updated.

**Review by jwoertink — COMMENTED** [review]
Looks like you'll just need to fix the merge conflict, but this is great. 👍

**Comment by nodeMD** [comment]
Thank you 🙂 There is also that link https://luckyframework.org/guides/ that redirects to: https://luckyframework.org/guides/getting-started/installing but that one looks like a reasonable one so I didn't updated it.

**Review by jwoertink — APPROVED** [review]
👍  Thank you so much!

**Pull request merged**